docs/en/html/Components/BLE/ble_stack/ble_stack.html

```diff
@@ -0,0 +1,971 @@
+<html class="writer-html5 translated-ltr" lang="en" style="height: 100%;"><head>
+  <meta charset="utf-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <title>17. BLE — BL602 IoT SDK release_bl_iot_sdk_1.6.11-1-g66bb28da document</title>
+  
+
+  
+  <link rel="stylesheet" href="../../../_static/css/theme.css" type="text/css">
+  <link rel="stylesheet" href="../../../_static/pygments.css" type="text/css">
+  <link rel="stylesheet" href="../../../_static/style.css" type="text/css">
+  <link rel="stylesheet" href="../../../_static/css/custom.css" type="text/css">
+
+  
+  
+  
+  
+
+  
+  <!--[if lt IE 9]>
+    <script src="../../../_static/js/html5shiv.min.js"></script>
+  <![endif]-->
+  
+    
+      <script type="text/javascript" id="documentation_options" data-url_root="../../../" src="../../../_static/documentation_options.js"></script>
+        <script src="../../../_static/jquery.js"></script>
+        <script src="../../../_static/underscore.js"></script>
+        <script src="../../../_static/doctools.js"></script>
+        <script src="../../../_static/language_data.js"></script>
+    
+    <script type="text/javascript" src="../../../_static/js/theme.js"></script>
+
+    
+    <link rel="index" title="index" href="../../../genindex.html">
+    <link rel="search" title="search for" href="../../../search.html">
+    <link rel="next" title="1. cronalarms" href="../../../API/sys/cronalarms.html">
+    <link rel="prev" title="16. Mesh" href="../mesh/mesh.html"> 
+<script>(function(){(function injection() {
+  var pageLang = 'zh-CN';
+  var userLang = 'en';
+
+  var uid = '1E07F158C6FA4460B352973E9693B329';
+  var teId = 'TE_' + uid;
+  var cbId = 'TECB_' + uid;
+
+  function show() {
+    window.setTimeout(function() {
+      window[teId].showBanner(true);
+    }, 10);
+  }
+
+  function newElem() {
+    var elem = new google.translate.TranslateElement({
+      autoDisplay: false,
+      floatPosition: 0,
+      multilanguagePage: true,
+      pageLanguage: pageLang
+    });
+    return elem;
+  }
+
+  if (window[teId]) {
+    show();
+  } else {
+    if (!window.google || !google.translate ||
+        !google.translate.TranslateElement) {
+      if (!window[cbId]) {
+        window[cbId] = function() {
+          window[teId] = newElem();
+          show();
+        };
+      }
+      var s = document.createElement('script');
+      s.src = 'https://translate.google.com/translate_a/element.js?cb=' +
```

Comment:
Are you just running the text through Google Translate?

Reply:
Yes, all the docs are translated via google. I feel it’s a good starting point till we get a linguist who knows Chinese and English!

If you’re curious about how I did it, I used the Google Translate extension for the job. To work, it injects scripts into the document; I cleaned up using my own little script (which unfortunately missed a tag in that document)

Reply:
I'm not so convinced this is a good idea. I discussed the translation I got from Google Translate when translating the Hello world example with a Chinese friend and it seems a fair bit of detail was lost in translation, even with something simple like that.

Reply:
Until we get someone who can translate between chinese and english, GTranslate is our only option